hm_pyhelper/hardware_definitions.py

```diff
@@ -91,7 +99,7 @@ def is_raspberry_pi() -> bool:
         },
 
     # Nebra Pi 0 Light Hotspot SPI Ethernet
-    'NEBHNT-LGT-ZS': {
+    'nebra-light1': {
         'FRIENDLY': 'Nebra Pi 0 Light Hotspot SE',
```

Comment:
I didn't like `0` as a part of the brand. We should use `Zero` name for both of the products.

Reply:
done